.github/workflows/build.yml

```diff
@@ -45,18 +77,19 @@ jobs:
     - name: Set up matrix of platforms and R versions
       id: setup-matrix
       run: |
-        platforms=$(python test/get_platforms.py ${{ github.event.inputs.platforms }})
+        platforms=$(python get_platforms.py ${{ github.event.inputs.platforms }})
         echo "platforms=$platforms" >> $GITHUB_OUTPUT
-        r_versions=$(python test/get_r_versions.py ${{ github.event.inputs.r_versions }})
+        r_versions=$(python get_r_versions.py ${{ github.event.inputs.r_versions }})
         echo "r_versions=$r_versions" >> $GITHUB_OUTPUT
 
   docker-images:
     needs: setup-matrix
     strategy:
       matrix:
         platform: ${{ fromJson(needs.setup-matrix.outputs.platforms) }}
-    runs-on: ubuntu-latest
-    name: Docker image (${{ matrix.platform }})
+        arch: [amd64, arm64]
+    runs-on: ${{ matrix.arch == 'amd64' && 'ubuntu-latest' || 'ubuntu-24.04-arm' }}
+    name: Docker image (${{ matrix.platform }}-${{ matrix.arch }})
     steps:
       - uses: actions/checkout@v3
```

Comment:
Should we consider using `actions/checkout@v4` instead?

Reply:
Done